src/hayhooks/server/app.py

```diff
@@ -147,10 +164,17 @@ def create_app() -> FastAPI:
         sys.path.append(additional_path)
         log.trace(f"Added {additional_path} to sys.path")
 
+    app_params: dict = {
+        "lifespan": lifespan,
+        "title": APP_TITLE,
+        "description": APP_DESCRIPTION,
+        "version": get_package_version_from_pypi("hayhooks"),
```

Comment:
Why the version is needed? Can we avoid downloading info from PyPI?

Reply:
This is to show the Hayhooks PyPI version in Swagger and Redocly. I agree that it's not the best to download from PyPI. A better alternative may be to have a `version.txt` file and update it when we do a release, but this requires a bit more effort. If you agree I would keep this for now (output of `get_package_version_from_pypi ` is cached) and improve it in another PR.

Reply:
ok please create an issue for that.